pkg/proxy/ipvs/proxier.go

```diff
@@ -108,6 +108,7 @@ var iptablesChains = []struct {
 	{utiliptables.TableNAT, KubeNodePortChain},
 	{utiliptables.TableNAT, KubeLoadBalancerChain},
 	{utiliptables.TableNAT, KubeMarkMasqChain},
+	{utiliptables.TableNAT, KubeMarkDropChain},
```

Comment:
Can we put this change in a separate commit - "flush and delete KUBE-MARK-DROP chain in IPVS clean up"

Reply:
ok